packages/oidc-http-server-pages/src/index.ts

```diff
@@ -0,0 +1,7 @@
+export { generateStaticPagesModule } from './create-static-pages';
```

Comment:
You could probably drop this export, `./create-static-pages` isn't something that consumers of this package should actually need